.github/workflows/quick.yaml

```diff
@@ -127,11 +127,17 @@ jobs:
           sudo sed --in-place -E 's/# (deb-src.*updates main)/  \1/g' /etc/apt/sources.list
           sudo apt-get --quiet=2 update
           sudo apt-get --quiet=2 build-dep squid
-          sudo apt-get --quiet=2 install linuxdoc-tools libtool-bin ${{ matrix.compiler.CC }}
+          sudo apt-get --quiet=2 install linuxdoc-tools libtool-bin ${{ matrix.compiler.CC }} ccache
 
       - name: Checkout sources
         uses: actions/checkout@v4
 
+      - name: Setup ccache
+        uses: squid-cache/ccache-action@v1.2.14
```

Comment:
@kinkie, if possible, please watch the upstream ccache-action repository for updates to reduce the chances that our new clone misses some useful ones. Thank you.

Reply:
Yes. I'll check it from time to time